user_manual/files/access_webdav.rst

```diff
@@ -366,6 +366,28 @@ Known Problems
   Known problems and their solutions are documented in the KB2123563_ article.
 
 **Problem**
+  Cannot map ownCloud as a WebDAV drive in Windows using self-signed certificate.
+
+**Solution**
+  1) Go to the your ownCloud instance via your favorite web browser ,
+  2) Click through until you get to the Certificate error in the browser status line.
+  3) View the cert, then from the Details tab, select Copy to File.
+  4) Save to the desktop as myOwnCloud.cer, for example,
```

Comment:
Perhaps this should say "Save to the desktop with an arbitrary name, for example, myOwnCloud.cer."